fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Avatar.kt

```diff
@@ -29,8 +29,9 @@ import com.microsoft.fluentui.theme.token.FluentIcon
 import com.microsoft.fluentui.theme.token.Icon
 import com.microsoft.fluentui.theme.token.controlTokens.*
 
-const val IMAGE_TEST_TAG = "Image"
-const val ICON_TEST_TAG = "Icon"
+// Tags used for testing
+const val AVATAR_IMAGE = "Image"
```

Comment:
add "Avatar " as prefix in value to avoid name classes while finding items in UI.